config.yml

```diff
@@ -516,7 +516,7 @@ steps:
       - type: removeBranchProtection
       - type: requestReviewFromRegistrant
       - type: respond
-        with: 17_protect-master.md
+        with: 17_protect-main.md
```

Comment:
no sure we want to change the response file name.  we would also need to change the file if we change it here in the config.